### DISCUSSION TITLE
repeat-mode unexpectedly sometimes enabled, and sometimes not

### DISCUSSION THREAD

**Discussion by shipmints** [minad/jinx#204] [Q&A]
I have two markdown files in the same directory with identical markdown mode and settings. The first, when entering jinx-correct mode, shows correction candidates for each word using M-n/M-p. The second one upon M-n/M-p shows repeat-mode advice, hiding correction candidates with "Repeat with ESC, n, p, $". I prefer avoiding repeat-mode advisory. I'm stumped on the cause of this behavioral discrepancy.

**Comment by minad** [comment]
I am not sure I understand the issue. Can you please provide a precise recipe based on emacs -Q, example files and your exact sequence of commands and key presses?

**Reply by shipmints** [reply]
I'll try.
